pkg/controllers/nodeclaim/lifecycle/launch.go

```diff
@@ -83,7 +83,8 @@ func (l *Launch) launchNodeClaim(ctx context.Context, nodeClaim *v1beta1.NodeCla
 		switch {
 		case cloudprovider.IsInsufficientCapacityError(err):
 			l.recorder.Publish(InsufficientCapacityErrorEvent(nodeClaim, err))
-			logging.FromContext(ctx).Error(err)
+			controllerruntime.LoggerFrom(ctx).Error(err, "")
```

Comment:
is there no syntax to log just the error?

Reply:
There was no syntax previously, but I can add it.

Reply:
Done.